css/src/components/gradient.scss

```diff
@@ -88,3 +91,24 @@ $border-gradient-direction: (
 	-webkit-text-fill-color: transparent;
 	line-height: 1.4; // required because of background clip, avoid "g" and "y" getting cut off
 }
+
+@mixin border-gradient($deg, $colorStart, $colorEnd) {
+	border-color: transparent !important;
+	background: linear-gradient(var(--theme-body-background) 0 0) padding-box,
```

Comment:
This is a restriction, `background-color-*` classes will not override.

Reply:
I think we're looking to add an atomic here, not a component. I'd recommend making changes in `atomics/border`

Reply:
Addressed feedback